<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1629</width>
    <height>999</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1629</width>
    <height>999</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="pushButton_close">
   <property name="geometry">
    <rect>
     <x>1570</x>
     <y>10</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelcamera">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>1041</width>
     <height>781</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_lizi">
   <property name="geometry">
    <rect>
     <x>1310</x>
     <y>468</y>
     <width>160</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;黑体&quot;;
color:rgb(242, 238, 255)</string>
   </property>
   <property name="text">
    <string>打开粒子特效</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_open">
   <property name="geometry">
    <rect>
     <x>1310</x>
     <y>160</y>
     <width>160</width>
     <height>80</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>243</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;黑体&quot;;
color:rgb(242, 238, 255)
</string>
   </property>
   <property name="text">
    <string>打开摄像头</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_closecamara">
   <property name="geometry">
    <rect>
     <x>1310</x>
     <y>775</y>
     <width>160</width>
     <height>80</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>243</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;黑体&quot;;
color:rgb(242, 238, 255)
</string>
   </property>
   <property name="text">
    <string>关闭摄像头</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
